src/interfaces/plug_keyring.ts

```diff
@@ -1,7 +1,16 @@
 import type PlugWallet from '../PlugWallet'
+import {
+    JSONWallet
+  } from '../interfaces/plug_wallet';
 
 export interface PlugState {
-    wallets: Array<PlugWallet>;
+    wallets: Array<JSONWallet>;
     password?: string;
     currentWalletId?: number;
+}
+
+export interface PlugStateInstance {
+  wallets: Array<PlugWallet>;
+  password?: string;
+  currentWalletId?: number;
 }
```

Comment:
New line at the end of the file

Reply:
dont we have a linter with these things?